openfl-tutorials/interactive_api/PyTorch_BrainMRI_UNet/envoy/shard_descriptor.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""BrainMRI Shard Descriptor"""
+
+import logging
+from typing import Any, Tuple
+import os
+import numpy as np
+from openfl.interface.interactive_api.shard_descriptor import ShardDataset
+from openfl.interface.interactive_api.shard_descriptor import ShardDescriptor
+import pandas as pd
+import torch
+import tarfile
+import gdown
+import nibabel as nib
+from tqdm import tqdm
+import sys
+import json
+
+
+logger = logging.getLogger(__name__)
+
+
+class brainMRIShard(ShardDataset):
+    """brainMRI Shard dataset class."""
+
+    def __init__(self, csv_path, rank=1, worldsize=1,
+                 crop_dims=None, dataset_dir='./',
+                 augment=True, device='cpu', set='train') -> None:
```

Comment:
```suggestion
                 augment=True, set='train') -> None:
```
Managing devices also is no concern for a Shard Descriptor